census/src/main/scala/hmda/census/model/TractLookup.scala

```diff
@@ -21,6 +21,7 @@ object TractLookup extends CbsaResourceUtils {
       val tractFips2010 = values(4)
       val minorityPopulation = Try(values(7).toDouble).getOrElse(0.0)
       val tractMfiToMsaPercent = Try(values(11).toDouble).getOrElse(100.0)
+      val medianYearHomesBuilt = 2015 - Try(values(12).toInt).getOrElse(0)
```

Comment:
There's a specific section in the report for "age unknown".... Are we sure we want to default a value to the 2015 year?

Reply:
Good call! I mistakenly thought that there were no Tract values that were missing a value for medianYearHomes built. Fixed in 994107f8f4b2a3c994f276f8c5d6bd22c8d46c9c.